<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>761</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №3 по Компьютерной графике</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>13</y>
      <width>417</width>
      <height>714</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="x1TextLabel">
         <property name="text">
          <string>          x1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="x1SpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="y1TextLabel">
         <property name="text">
          <string>          y1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="y1SpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="x2TextLabel">
         <property name="text">
          <string>          x2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="x2SpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="y2TextLabel">
         <property name="text">
          <string>          y2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="y2SpinBox">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="fgLabel">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="fgPushButton">
         <property name="text">
          <string>Выбрать цвет линии</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="setBGColorToFGPushButton">
       <property name="text">
        <string>Линию в цвет фона</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setDefaultFGColorPushButton">
       <property name="text">
        <string>Цвет линии по умолчанию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawLinePushButton">
       <property name="text">
        <string>Нарисовать линию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ddaRadioButton">
       <property name="text">
        <string>ЦДА</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bresenhamFloatRadioButton">
       <property name="text">
        <string>Брезенхем</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bresenhamIntegerRadioButton">
       <property name="text">
        <string>Брезенхем (одна интенсивность)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bresenhamAntialiasedRadioButton">
       <property name="text">
        <string>Брезенхем с устранением ступенчатости</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="wuRadioButton">
       <property name="text">
        <string>Ву</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="defaultQtRadioButton">
       <property name="text">
        <string>Стандартный алгоритм (Qt)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lengthTextLabel">
         <property name="text">
          <string>Длина</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="lengthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="angleTextLabel">
         <property name="text">
          <string>Угол</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="angleSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>89</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="drawSunPushButton">
       <property name="text">
        <string>Нарисовать солнце</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearAllPushButton">
       <property name="text">
        <string>Очистить всё</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statisticsLadderPushButton">
       <property name="text">
        <string>Статистика (количество ступеней)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statisticsPushButton">
       <property name="text">
        <string>Статистика (время отрисовки)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>725</width>
      <height>725</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>725</width>
      <height>725</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
